apps/changelog/classes/change_log.py

```diff
@@ -0,0 +1,82 @@
+# Python Standard Library Imports
+from collections import namedtuple
+
+
+# isort: off
+
+
+class ChangeLog(
+    namedtuple(
+        'ChangeLog',
+        'origin_url,log_entries',
+    )
+):
```

Comment:
# Reviewer Note
Python 2 compatible code to achieve `@dataclass` decorator in Python 3